ssl/ssl_lib.c

```diff
@@ -6942,3 +7035,142 @@ int SSL_CTX_set0_tmp_dh_pkey(SSL_CTX *ctx, EVP_PKEY *dhpkey)
     ctx->cert->dh_tmp = dhpkey;
     return 1;
 }
+
+/* QUIC-specific methods which are supported on QUIC connections only. */
+int SSL_tick(SSL *s)
+{
+    SSL_CONNECTION *sc;
+#ifndef OPENSSL_NO_QUIC
+    QUIC_CONNECTION *qc = QUIC_CONNECTION_FROM_SSL(s);
+
+    if (qc != NULL)
+        return ossl_quic_tick(qc);
+#endif
+
+    sc = SSL_CONNECTION_FROM_SSL_ONLY(s);
+    if (sc != NULL && SSL_CONNECTION_IS_DTLS(sc))
+        return DTLSv1_handle_timeout(s);
```

Comment:
`SSL_tick` is documented as returning 1 on success or 0 on error - but `DTLSv1_handle_timeout` seems to return 1 on success, 0 if the timer wasn't expired yet, or the SSL object is NULL or not a DTLS object, and -1 on other errors.